app/templates/orders/view.hbs

```diff
@@ -19,41 +19,35 @@
         @data={{this.model.order}}
         @event={{this.model.event}}
         @eventCurrency={{this.model.order.event.paymentCurrency}} />
-    </div>
-    <div class="mobile hidden six wide column">
-      <Orders::EventInfo
-        @data={{this.model.order}} />
-    </div>
-  </div>
-  <div class="row">
-    <div class="column right aligned">
-      <button class="ui labeled icon {{if this.isLoadingTickets 'loading'}} {{if this.device.isMobile 'fluid'}} button"
-        {{action this.downloadTickets this.model.event.name this.model.order.identifier}}>
-        <i class="ticket alternate icon"></i>
-        {{t 'Download Tickets'}}
-      </button>
-      <button {{action 'downloadInvoice' this.model.event.name this.model.order.identifier }}
-        class="ui labeled icon blue {{if this.isLoadingInvoice 'loading'}} {{if this.device.isMobile 'fluid'}} button">
-        <i class="download icon"></i>
-        {{t 'Download Invoice'}}
-      </button>
-    </div>
-  </div>
-  <div class="row">
-    <div class="ten wide column">
+      <br/>
       <Forms::Orders::AttendeeList
         @save="save"
         @data={{this.model.order}}
         @fields={{this.model.form}}
         @event={{this.model.event}} />
     </div>
     <div class="six wide column">
-      <div class="row">
-        <div class="mobile hidden row">
-          <Orders::TicketHolder
-            @data={{this.model.order}} />
-        </div>
+      <Orders::EventInfo
+        @data={{this.model.order}} />
+      <br/>
+      <div class="{{if (eq this.device.isMobile false) 'display-flex'}}">
```

Comment:
Why do you need to check that it is not mobile?

Reply:
@iamareebjamal sir, if we implement this class at mobile device, buttons are not stacking on one another.

![explain 1](https://user-images.githubusercontent.com/72552281/98758585-22a30200-23f5-11eb-9cfe-86647577026b.png)

Reply:
Looks good, no need for stacking